rrmngmnt/filesystem.py

```diff
@@ -54,3 +55,51 @@ def touch(self, file_name, path):
         """
         full_path = os.path.join(path, file_name)
         return self.host.run_command(['touch', full_path])[0] == 0
+
+    def mkdir(self, path):
+        """
+        Create directory on host
+
+        :param path: directory path
+        :type path: str
+        :return: True, if action succeed, otherwise False
+        :rtype: bool
+        """
+        return self.host.run_command(
+            ['mkdir', path]
+        )[0] == 0
+
+    def chown(self, path, username, groupname):
+        """
+        Change owner of file or directory
+
+        :param path: file or directory path
+        :type path: str
+        :param username: change user owner to username
+        :type username: str
+        :param groupname: change group owner to groupname
+        :type groupname: str
+        :return: True, if action succeed, otherwise False
+        :rtype: bool
+        """
+        uid, gid = (
+            pwd.getpwnam(owner).pw_uid for owner in (username, groupname)
```

Comment:
what if I want to manipulate, user which doesn't exist on local machine, but it does on remote machine ...